--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -4,7 +4,7 @@
     alias(libs.plugins.ksp)
     alias(libs.plugins.hilt)
     alias(libs.plugins.sentry)
-    alias(libs.plugins.google.services)
+    // alias(libs.plugins.google.services)
     alias(libs.plugins.compose.compiler)
 }

@@ -13,11 +13,22 @@
     namespace = "au.com.shiftyjelly.pocketcasts"

     defaultConfig {
-        applicationId = project.property("applicationId").toString()
+        applicationId = "unlocked.premium.pocketcasts"
         targetSdk = project.property("targetSdkVersion") as Int
         multiDexEnabled = true
     }

+    signingConfigs {
+        create("release") {
+            storeFile = file("../debug.keystore")
+            storePassword = "android"
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+        }
+    }
+
     sourceSets {
         getByName("androidTest") {
             assets.srcDir(files("$rootDir/modules/services/model/schemas"))
@@ -45,6 +56,7 @@

         named("release") {
             manifestPlaceholders["appIcon"] = "@mipmap/ic_launcher"
+            signingConfig = signingConfigs.getByName("release")

             if (!file("${project.rootDir}/sentry.properties").exists()) {
                 println("WARNING: Sentry configuration file 'sentry.properties' not found. The ProGuard mapping files won't be uploaded.")
